creator-node/src/services/stateMachineManager/makeOnCompleteCallback.js

```diff
@@ -1,7 +1,7 @@
 const { logger: baseLogger, createChildLogger } = require('../../logging')
 const { QUEUE_NAMES, JOB_NAMES } = require('./stateMachineConstants')
 const {
-  MetricRecordType
+  METRIC_RECORD_TYPE
```

Comment:
this and everything else below are variable name changes + styling. no functionality changes